tests/ReplayData/SubIssue.setUp.txt

```diff
@@ -0,0 +1,21 @@
+
+https
```

Comment:
With `self.g.get_repo("PyGithub/PyGithub", lazy=True)`, this call should disappear.
```suggestion
https
```